client/state/happychat/middleware.js

```diff
@@ -148,6 +156,10 @@ export default function( connection = null ) {
 				connectChat( connection, store );
 				break;
 
+			case HAPPYCHAT_INITIALIZE:
```

Comment:
As we plan to decouple Happychat from Calypso we'll want to make sure we don't hook into any Calypso actions. The idea here is that `HAPPYCHAT_INITIALIZE` will be sent by the platform containing the Happychat client to notify that it should do any prep work. Right now this is still coupled as `client/boot/project/wordpress-com.js` is dispatching the action directly, but longer term we may provide an imperative command that dispatches this action, or give host platforms some hooks directly into the Happychat Redux system. /cc @beaucollins